UI/ClientUI.h

```diff
@@ -8,7 +8,9 @@
 #include <boost/date_time/posix_time/posix_time.hpp>
 
 #include "SaveFileDialog.h"
-#include "../universe/EnumsFwd.h"
+#include "../universe/Enums.h"
```

Comment:
universe/EnumsFwd.h was a lot lighter than universe/Enums.h is. Can the forward declare only alternative be retained?

Reply:
Restored it back.